monitoring_log_requests/models/ir_http.py

```diff
@@ -62,8 +62,10 @@ def _monitoring_info(cls, request, response, begin, end):
             # response things
             "response_status_code": None,
         }
-        if hasattr(request, "status_code"):
-            info["status_code"] = response.status_code
+        if isinstance(response, Response) and hasattr(response, "status_code"):
+            info["response_status_code"] = response.status_code
+        else:
```

Comment:
Can this not be misleading ? Or are we always sure that an inexistent status_code represents a 200 status code meaning that the request was successful :thinking: